src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -18,17 +18,28 @@ import { ReactionPage } from "components/templates/ReactionPage";
 import { ChatSidebar } from "components/organisms/ChatSidebar";
 import { UserProfileModal } from "components/organisms/UserProfileModal";
 import { WithNavigationBar } from "components/organisms/WithNavigationBar";
-
+import { useWindowDimensions } from "hooks/useWindowDimensions";
+import { setChatSidebarVisibility } from "store/actions/Chat";
 import { AnnouncementMessage } from "components/molecules/AnnouncementMessage";
+import { LARGE_SCREEN_WIDTH } from "settings";
+import { useDispatch } from "hooks/useDispatch";
 
 export interface TemplateWrapperProps {
   venue: AnyVenue;
 }
 
 const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
+  const { width } = useWindowDimensions();
+  const dispatch = useDispatch();
   const history = useHistory();
   const match = useRouteMatch();
 
+  useEffect(() => {
+    if (width > LARGE_SCREEN_WIDTH) {
+      dispatch(setChatSidebarVisibility(true));
+    }
+  }, [dispatch, width]);
+
```

Comment:
this logic should be moved into a hook. Probably into chats hook.

Reply:
got it

Reply:
done